### ISSUE TITLE
Fork.dev keeps asking for Bitbucket credentials

### ISSUE THREAD

**Issue by robert-niestroj** [ForkIssues/TrackerWin#1777]
Hi, after the latest update of Fork to 1.81.1.0 and the update of the internal git instance Bitbucket Server dialogs keep popping up everytime fork want's to do something git related. If worked fine. 
As a workaround i installed Git 2.33 and chose this instead of the internal Fork Git instance and it works with this version. 
Please fix so that it works with the internal git instance. 
Verified also on colleagues PC. Same behavior, same workaround works.
Windows 10 22H2

**Comment by DanPristupov** [comment]
> As a workaround i installed Git 2.33

Does it work properly with the latest Git 2.39.1? I'm asking because git-credential-manager is a part of the official git-for-windows distribution and is not related to Fork. It might have a problem 2.39.1.

How did you log in to BitBucket? Did you use File -> Accounts -> BitBucket?

**Comment by DanPristupov** [comment]
I just logged out of my bitbucket account in Fork and authenticated using git-credential-manager. We'll see if I manage to reproduce the problem myself.

**Comment by robert-niestroj** [comment]
It does not work with git 2.39.1 installed standalone. I had a fresh OS, installed git 2.39.1 and Fork latest version. Everything worked. Then on Sunday i think came the Fork update and after that it's stopped working - the credential windows started appearing. I tried a few thing without success. Today colleague told me that it works for him with 2.32 so i installed 2.33 and it worked. 
We use a self-hosted version on bitbucket i think if that matters.

**Comment by DanPristupov** [comment]
OK, I'll check 'BitBucket Server'.

Did you try to log in to your BitBucket Server using File -> Accounts -> BitBucket Server?

**Comment by robert-niestroj** [comment]
> Did you try to log in to your BitBucket Server using File -> Accounts -> BitBucket Server?

No. I don't have this token. Just Username/Password.

**Comment by DanPristupov** [comment]
OK, I tried with BitBucket server and it worked for me.

Here are steps you should check:

1. Ensure Fork uses the internal git instance 2.39.1 (File -> Preferences -> Git)
2. Ensure that credential manager in `%localappdata%\Fork\gitInstance\2.39.1\etc\gitconfig` is configured like following:
```
[credential]
	helper = manager
```
3. Open Credential Manager application (you can find it in the Windows start menu). Open the `Windows Credentials` tab and delete the entry which looks like `git:https://yourserver`.
4. Try to fetch in Fork and enter your credentials in the BitBucket dialog. Confirm that a new entry has been created in Credential Manager (press F5 to refresh it).

**Comment by robert-niestroj** [comment]
Yes. This steps do work. The new entry is created. But the next fetch triggers this BitBucket window. He seemingly cannot read the values from the credentail manager.

I cancelled the credentials window and got this response in Fork: 
`
fatal: helper error (-1): User cancelled dialog.
error: unable to read askpass response from 'C:\Users\3797\AppData\Local\Fork\app-1.81.1\Fork.AskPass.exe'
bash: line 1: /dev/tty: No such device or address
error: failed to execute prompt script (exit code 1)
fatal: could not read Username for 'https://thisismy.bitbucketserver.com': No such file or directory
`


Got also this in the logs: 
`⚠️ 2023-01-31 18:57:53.472 Cannot parse askpass request 'Username for 'https://thisismy.bitbucketserver.com': '`
